compiled_starters/sqlite-starter-swift/Sources/swift-sqlite-challenge/Main.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+@main
+struct Main {
+
```

Comment:
@Terky quick question - is it intentional to leave an extra space below the `struct Main {` line here? Same for line 40 (`extension UnsignedInteger {`)? 

Is this is a widely accepted code style for Swift? Just curious, haven't seen this elsewhere.